README.md

```diff
@@ -179,8 +180,8 @@ Want more testing? ▶ [see the Wiki](https://github.com/qdm12/private-internet-
 
     | Variable | Default | Choices | Description |
     | --- | --- | --- | --- |
-    | 🏁 `USER` | | | Your username |
-    | 🏁 `PASSWORD` | | | Your password |
+    | 🏁 `USER` | | | Your **service** username. Can be found at the bottom of the [manual setup](https://account.surfshark.com/setup/manual) page |

```

Comment:
```suggestion
    | 🏁 `USER` | | | Your **service** username, found at the bottom of the [manual setup page](https://account.surfshark.com/setup/manual) |
```

Just being a short line maniac, forgive me 😄

Reply:
Normally my editor would have done that for me. The github editor is rather limited though ;)

Reply:
Your editor simplifies sentences?! I'm listening

Reply:
Let me rephrase, normally I would have noticed the long lines and would have tried to simplify ;) I have a big red line at 80 characters. Although when it comes to style guides, pre-commit might be useful: https://pre-commit.com/

Reply:
Ah ok that makes sense! I used some linters for my code telling me to make lines shorter but it's really a case to case. Especially in the readme, the line length doesn't really matter. It's more of a human phrase cognitive complexity let's say. And that silly 25KB limit :smile: